assets/payment/settings_script.js

```diff
@@ -3,19 +3,25 @@
 let BuyLicenseWindow = null;
 let UpdatePlanWindow = null;
 let CancelSubscriptionWindow = null;
+let CustomFormWindow = null;
 let wait = false;
 /* eslint-disable */
 function redirectToCallback(param) {
 	window.location.href += "&" + param;
 }
 function cancelPayment(){
-	BuyLicenseWindow.close();
+	CustomFormWindow.close();
 	showBuyLicenseWindow();
 }
 function confirmPlanChange(){
 	BuyLicenseWindow.close();
 	confirmPlanChangeWindow();
 }
+//this function it's used by dialog CustomFormWindow 
+function customStripeForm(planRank){
```

Comment:
Codacy found an issue: ['customStripeForm' is defined but never used.](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5864152)